app/cronjob/trailingTrade/step/determine-action.js

```diff
@@ -27,7 +51,11 @@ const canBuy = data => {
     buy: { currentPrice: buyCurrentPrice, triggerPrice: buyTriggerPrice }
   } = data;
 
-  return buyCurrentPrice <= buyTriggerPrice && currentGridTrade !== null;
+  return (
+    buyCurrentPrice <= buyTriggerPrice &&
+    currentGridTrade !== null &&
+    !isLowerThanStopLossTriggerPrice(data)
```

Comment:
Can we add tests for this change? 😄

Reply:
Yes, of course. Will you do?

Reply:
Sure. When I have time, I will update tests. :)